e2e/e2e.js

```diff
@@ -29,15 +29,67 @@ test('sdk init test', async () => {
 });
 
 test('test getActivities', async () => {
-  //check success response
-  // var res = await sdkClient.getXTActivityById(321)
   var res = await sdkClient.getActivities()
-  expect(res.total).toBe(9)
+  expect(res.total).toBe(10)
 })
 
 test('test getABActivityById', async () => {
-  //check success response
   var res = await sdkClient.getABActivityById(204051)
```

Comment:
this would fail if the user's test environment had different activities in their database. You should only get the activity that you specifically created for the test. The e2e test should create an activity for testing, then delete it after.